packages/ui/src/components/Radio/index.tsx

```diff
@@ -236,16 +242,23 @@ export const Radio = forwardRef(
               <RadioMarkedIcon />
             </Ring>
             {label ? (
-              <StyledLabel htmlFor={`${computedName}-${value}`}>
+              <>
                 {typeof label === 'string' ? (
-                  <Text as="label" variant="body" prominence="default">
+                  <StyledTextLabel
+                    as="label"
+                    variant="body"
+                    prominence="default"
+                    htmlFor={`${computedName}-${value}`}
```

Comment:
useId ?

Reply:
there is a risks to break other stuff I think, computed name uses `useId()`